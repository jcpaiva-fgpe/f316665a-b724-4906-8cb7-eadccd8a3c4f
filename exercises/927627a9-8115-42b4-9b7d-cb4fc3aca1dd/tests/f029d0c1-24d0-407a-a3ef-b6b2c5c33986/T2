clap your hands
it has beaten palms
