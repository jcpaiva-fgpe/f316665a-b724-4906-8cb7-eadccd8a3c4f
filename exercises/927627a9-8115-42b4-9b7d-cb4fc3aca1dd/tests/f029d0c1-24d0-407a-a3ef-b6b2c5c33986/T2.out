clap your hands
clap your hands
